tests/src/test/scala/whisk/core/admin/WskAdminTests.scala

```diff
@@ -91,4 +91,37 @@ class WskAdminTests
         val ns = wsk.namespace.whois()
         wskadmin.cli(Seq("user", "get", ns)).stdout.trim should be(wskprops.authKey)
     }
+
+    it should "block and unblock a user respectively" in {
+        val wskadmin = new RunWskAdminCmd {}
+        val auth = AuthKey()
+        val subject = Subject().asString
+        try {
+            println(s"CRD subject: $subject")
+            wskadmin.cli(Seq("user", "create", subject, "-u", auth.compact))
+
+            whisk.utils.retry({
+                // reverse lookup by namespace
+                wskadmin.cli(Seq("user", "list", "-k", subject)).stdout.trim should be(auth.compact)
+            }, 10, Some(1.second))
+
+            // block the user
+            wskadmin.cli(Seq("user", "block", subject))
+
+            // wait until the user can no longer be found
+            whisk.utils.retry({
+                wskadmin.cli(Seq("user", "list", subject)).stdout.trim should startWith("no identities")
```

Comment:
this behavior might be a little confusing - i understand why it's done this way: requires no schema changes but maybe in the wskadmin utility, listing namespaces for a blocked subject could at least tell you the identities are blocked... this could be helpful if for example an identity is blocked, but without knowing so, someone tries to add new namespaces and end up "startled" in that the addition will work but listing the spaces will return "no identities".